CONTRIBUTING.md

```diff
@@ -32,6 +32,8 @@ Clone the repo and run the following in the terminal/command line/powershell:
 
 ```bash
 npm install
+npm run prepare
+npm run build
```

Comment:
I would think `npm run build` is not required to do `npm start`?

Reply:
Makes sense to me to build before running :P
You are right, it isn't actually a required step to run `build` before running `start` in order to see the default test page. However, many of the samples require the presence of the /dist folder to work, and that may not be obvious to new contributors.